commons/api/src/bundle/commons_es.properties

```diff
@@ -1,6 +0,0 @@
-# this defines the entity description for the commons provider
-commons=<br />Proporciona un conjunto de puntos finales RESTful para recuperar publicaciones de COMMONS. <br /><br />
```

Comment:
What happens with the _es translation? I see that commons_es and Messages_es have been removed.

Reply:
That's no good. I'll check that.